specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/configurations.json

```diff
@@ -32,13 +32,56 @@
         }
     },
     "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}/configurations": {
+            "post": {
+                "tags": [
+                    "Configurations"
+                ],
+                "operationId": "Configurations_List",
+                "description": "Gets all configuration information for an HDI cluster.",
+                "x-ms-examples": {
+                    "Get all configuration information": {
+                        "$ref": "./examples/HDI_Configurations_List.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ClusterNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "./operations.json#/definitions/ErrorResponse"
+                        }
+                    },
+                    "200": {
+                        "description": "OK response definition.",
+                        "schema": {
+                            "$ref": "#/definitions/ClusterConfigurations"
+                        }
+                    }
+                }
+            }
+        },
         "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}/configurations/{configurationName}": {
             "post": {
                 "tags": [
                     "Configurations"
                 ],
-                "operationId": "Configurations_UpdateHTTPSettings",
-                "description": "Configures the HTTP settings on the specified cluster.",
+                "operationId": "Configurations_Update",

```

Comment:
This would be a breaking change from the SDK's perspective. Since the changes are still in preview we should be fine.